### ISSUE TITLE
Get Prost-built code auto-generating in Bazel

### ISSUE THREAD

**Issue by daviddrysdale** [project-oak/oak#850]
#801 checked in the Prost-generated Rust code for our protobufs, but this should be generated on the fly as part of a Bazel rule.  

[Details here](https://github.com/project-oak/oak/pull/801#issuecomment-613557625)

**Comment by daviddrysdale** [comment]
No longer relevant after #1016.
